crypto/impl_dispatch_test.cc

```diff
@@ -61,7 +66,12 @@ class ImplDispatchTest : public ::testing::Test {
 #else
         false;
 #endif // MY_ASSEMBLER_IS_TOO_OLD_FOR_512AVX
-#endif  // X86 || X86_64
+#elif defined(OPENSSL_AARCH64)
+    aes_hw = CRYPTO_is_ARMv8_AES_capable();
+    aes_vpaes = CRYPTO_is_NEON_capable();
+    armv8_gcm_pmull_ = CRYPTO_is_ARMv8_PMULL_capable();
```

Comment:
nit: can we stick to `aes_*` naming for these two (without `armv8`) for consistency?